src/util.ts

```diff
@@ -168,6 +168,22 @@ export function formatAsUTCISO(
   return resultString;
 }
 
+/**
+ * Examines the runtime environment and returns the appropriate tracking string.
+ * @returns {string} metrics tracking string based on the current runtime environment.
+ */
+export function getRuntimeTrackingString(): string {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  if (globalThis.Deno) {
```

Comment:
I believe looking up the specific deno version property here might help avoid any false positives:

```suggestion
  if (globalThis.Deno && globalThis.Deno.version && globalThis.Deno.version.deno) {
```